hw3/01-characters.js

```diff
@@ -1,4 +1,27 @@
-// url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+async function displayCharacterCards(data) {
+  const sectionTagHTML = document.querySelector('section');
+  data.forEach((character) => {
+    const characterCards = `
+      <div class="card">
+        <img src="${character.imageUrl}">
```

Comment:
I ran your code on my system and provided some feedback below. Overall, it looks good. 
I have focused my feedback on part 1 and 2 since these are the parts I understand the most.

Regarding 1:
- Running a WAVE check on it reveals 53 errors, all of which could easily be fixed by adding alt text to the images.
- Looking at the code, why are the name and title in heading 5 and heading 6 respectively? Headings 2-4 appear to have been skipped, and this comes up as an alert in WAVE.

Regarding 2:
- No changes necessary, just noticed the difference in granularity between yours and mine. Interesting the way you grouped many families together as other.